sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/placement/PassthroughRule.java

```diff
@@ -20,8 +20,8 @@ public PassthroughRule() {
     }
 
     @Override
-    public Offer filter(Offer offer, OfferRequirement offerRequirement, Collection<TaskInfo> tasks) {
-        return offer;
+    public EvaluationOutcome filter(Offer offer, OfferRequirement offerRequirement, Collection<TaskInfo> tasks) {
+        return EvaluationOutcome.pass(this, "Rubber stamp approval");
```

Comment:
Extreme nit: maybe change the wording for the pass reason to be more plainly descriptive? I could see this being a strange phrase to encounter for a non-native English speaker looking through logs.